modules/core/src/main/java/org/apache/fluo/core/impl/TransactionImpl.java

```diff
@@ -293,6 +275,28 @@ public Bytes get(Bytes row, Column column) {
     return ret;
   }
 
+  private Map<RowColumn, Bytes> getImpl(Collection<RowColumn> rowColumns,
+      Map<Bytes, Set<Column>> locksSeen) {
```

Comment:
`locksSeen` should be named `readLocksSeen`

Reply:
Updated to remove param. Nothing ever calls this method with anything but the instance variable `readLocksSeen` so this method now directly uses that object.